src/ui/components/SearchInput/index.js

```diff
@@ -73,7 +93,7 @@ export default class SearchInput extends React.Component {
   }
 
   render() {
-    const { className, name, placeholder, ...props } = this.props;
```

Comment:
You should double check every component that was using `SearchInput` previously to make sure you expanded everything that was in use.